hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CreateTableProcedure.java

```diff
@@ -336,41 +335,21 @@ private void postCreate(final MasterProcedureEnv env)
       final TableDescriptor tableDescriptor, List<RegionInfo> newRegions,
       final CreateHdfsRegions hdfsRegionHandler) throws IOException {
     final MasterFileSystem mfs = env.getMasterServices().getMasterFileSystem();
-    final Path tempdir = mfs.getTempDir();
 
     // 1. Create Table Descriptor
     // using a copy of descriptor, table will be created enabling first
-    final Path tempTableDir = CommonFSUtils.getTableDir(tempdir, tableDescriptor.getTableName());
+    final Path tableDir = CommonFSUtils.getTableDir(mfs.getRootDir(), tableDescriptor.getTableName());
```

Comment:
We do not need to delete or clean the old tableDir if exists first?

Reply:
We alerady call `DeleteTableProcedure.deleteFromFs(env, getTableName(), newRegions, true)` on line #104 before we call `createFsLayout`.

Reply:
OK.